modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/controller/AccountController.java

```diff
@@ -64,7 +65,7 @@ public AccountController(AuthenticationManager authMgr,  AccountsService account
      * @param user User.
      */
     @GetMapping(path = "/api/v1/user")
-    public ResponseEntity<UserResponse> user(@AuthenticationPrincipal UserDetails user) {
+    private ResponseEntity<UserResponse> user(@AuthenticationPrincipal UserDetails user) {
```

Comment:
Why you change visibility of methods annotated by RequestMapping?

Reply:
It is not used anywhere. Why to have it public?

Reply:
Guys from Spring team do not recommend of using private methods https://jira.spring.io/projects/SPR/issues/SPR-16878